swc/defs.bzl

```diff
@@ -30,6 +31,17 @@ attribute.
     toolchains = _swc_lib.toolchains,
 )
 
+def _is_file_present(label):
```

Comment:
already have this in bazel-lib: `file_exists`

Reply:
ah, should use that in rules_ts too then. I added that didn't I 💇🏻